package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "publiclab-editor",
-  "version": "2.1.1",
+  "version": "2.1.2",
```

Comment:
OK! And once this merges, can you create a new Release with the release notes here? I think we need to add a few more updates from https://github.com/publiclab/PublicLab.Editor/pulse as well to summarize other changes from folks. @publiclab/editor who else has had changes merged? Can we include https://github.com/publiclab/PublicLab.Editor/pull/568 @keshav234156 ?

Reply:
yes, we can include the #568. 
Also, I have updated the list have a look.